woodwork/utils.py

```diff
@@ -442,3 +442,18 @@ def _update_progress(start_time, current_time, progress_increment,
         callback_function(progress_increment, new_progress, total, unit, elapsed_time)
 
         return new_progress
+
+
+def _infer_datetime_format(dates, n=100):
```

Comment:
Just wondering if 100 rows is a large enough default? For datasets with tens of thousands of rows, using only 100 rows seems like a really small sample. We can always leave this as it is for now and update later if it becomes a problem though.

Reply:
set the default size to be 10% of the rows!